gpjax/pytree.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 The JaxLinOp Contributors. All Rights Reserved.
+#
```

Comment:
Perhaps we should be thinking of `JaxGaussianProcesses` or `GPJax` contributors.